dxa-framework/dxa-common-impl/src/test/java/com/sdl/webapp/common/impl/markup/MarkupTest.java

```diff
@@ -9,26 +9,126 @@
 import com.sdl.webapp.common.api.mapping.semantic.annotations.SemanticPropertyInfo;
 import com.sdl.webapp.common.api.mapping.semantic.config.SemanticVocabulary;
 import com.sdl.webapp.common.api.model.entity.AbstractEntityModel;
+import com.sdl.webapp.common.api.model.region.RegionModelImpl;
+import com.sdl.webapp.common.exceptions.DxaException;
 import com.sdl.webapp.common.markup.Markup;
+import org.joda.time.DateTime;
+import org.joda.time.format.DateTimeFormat;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
 import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
 import org.springframework.test.context.support.AnnotationConfigContextLoader;
 
 import java.lang.reflect.Field;
 import java.util.LinkedHashSet;
 import java.util.Locale;
 import java.util.Set;
 
+import static org.hamcrest.CoreMatchers.is;
+import static org.junit.Assert.assertThat;
 import static org.mockito.Mockito.mock;
 import static org.mockito.Mockito.when;
 
 /**
  * Unit tests for {@code Markup}.
  */
+@RunWith(SpringJUnit4ClassRunner.class)
 @ContextConfiguration(loader = AnnotationConfigContextLoader.class)
 public class MarkupTest {
```

Comment:
Thank you for writing tests. Change please tests to BDD style/naming as pure TDD naming is harder to read through and understand the purpose of the test